### DISCUSSION TITLE
How to add new custom table within Elsa in v3 version?

### DISCUSSION THREAD

**Discussion by sagargolu** [elsa-workflows/elsa-core#4587] [Q&A]
Hello everyone,

I want to add one custom table within ElsaDBContext. I am adding it but somehow at the time of database creation it is not populating. So would you please guide us how to add it? I am using SQLServer.

**Comment by sfmskywalker** [comment]
Hey there, I haven't tried adding more tables myself, but I noticed an interesting approach by [this project](https://github.com/appany/AppAny.Quartz.EntityFrameworkCore.Migrations) that seems to be able to add DB sets to a given DB context. I haven't dug deep, so I'm not sure.

Alternatively to extending the ElsaDbContext, you can always create your own, with its own migrations etc - while using the same DB connection.
